tweepy/models.py

```diff
@@ -207,7 +207,7 @@ class DirectMessage(Model):
     @classmethod
     def parse(cls, api, json):
         dm = cls(api)
-        for k, v in json.items():
+        for k, v in pyjson.loads(dm._api.last_response.text).items():
```

Comment:
I'm not sure what the reasoning is in getting the JSON response in such a roundabout way.